clouds/bigquery/modules/doc/h3/H3_POLYFILL.md

```diff
@@ -6,14 +6,9 @@ H3_POLYFILL(geography, resolution)
 
 **Description**
 
-Returns an array with all the H3 cell indexes **with centers** contained in a given polygon. It will return `null` on error (invalid geography type or resolution out of bounds). In case of lines, it will return the H3 cell indexes intersecting those lines. For a given point, it will return the H3 index of cell in which that point is contained.
+Returns an array with all the H3 cell indexes which intersect a given polygon, line or point. It will return `null` on error (invalid geography type or resolution out of bounds).
 
-````hint:info
-**warning**
-
-Lines polyfill is calculated by approximating S2 cells to H3 cells, in some cases some cells might be missing.
-
-````
+This function is equivalent to using [`H3_POLYFILL_MODE](h3#h3_polyfill) with mode `intersects`. If th input geometry is a polygon check that function for more options and better performance.
```

Comment:
Is the link correct? 
also it looks like there is a small typo in "If the input geometry".

Reply:
fixed type, but not shure what should be the link... cheking

Reply:
Yes, it will be replaced by the final link in the script (gitbook-documentation)